README.md

```diff
@@ -46,3 +46,17 @@ Transfer the apk to your device and install it to use the application.
 
 ## Note 
 * Marshmallow runtime permissions support not added. You need to enable permission manually in app settings for this app to work
+
+## Contributors
+[![](https://sourcerer.io/fame/arshadkazmi42/arshadkazmi42/android-contacts-scifiui/images/0)](https://sourcerer.io/fame/arshadkazmi42/arshadkazmi42/android-contacts-scifiui/links/0)
+[![](https://sourcerer.io/fame/arshadkazmi42/arshadkazmi42/android-contacts-scifiui/images/1)]
```

Comment:
Something looks broken here. I think the `line break` added here is breaking it.
Can you remove the the line breaks. Or copy paste the exact code as given in the issue and paste it here and update the PR

Reply:
Oh, okay. Will do it  .